server/controllers/getEvents.js

```diff
@@ -1,12 +1,10 @@
 const { Event } = require('../database/models');
 
-exports.getEvents = ((req, res, next) => {
+exports.getEvents = ((req, res) => {
```

Comment:
the point of adding id to the endpoint as you did in router file **/events/:id** to use it here as condition into query so it will only get back the events that related to the service with this **id**.
```
where serviceId = id
```